tests/test-cases/Group6-VIC-Machine/6-07-Create-Network.robot

```diff
@@ -329,13 +324,45 @@ Bridge network - valid with IP range
     Pass execution  Test not implemented
 
 Container network invalid 1
-    Pass execution  Test not implemented
+    Set Test Environment Variables
+    # Attempt to cleanup old/canceled tests
+    Run Keyword And Ignore Error  Cleanup Dangling VMs On Test Server
+    Run Keyword And Ignore Error  Cleanup Datastore On Test Server
+
+    ${output}=  Run  bin/vic-machine-linux create --name=%{VCH-NAME} --target="%{TEST_USERNAME}:%{TEST_PASSWORD}@%{TEST_URL}" --thumbprint=%{TEST_THUMBPRINT} --image-store=%{TEST_DATASTORE} --bridge-network=bridge --container-network 'VM Network With Spaces' ${vicmachinetls}
+    Should Contain  ${output}  A network name must be supplied for port group VM Network With Spaces
+    Should Contain  ${output}  vic-machine-linux create failed
 
 Container network invalid 2
-    Pass execution  Test not implemented
+    Set Test Environment Variables
+    # Attempt to cleanup old/canceled tests
+    Run Keyword And Ignore Error  Cleanup Dangling VMs On Test Server
+    Run Keyword And Ignore Error  Cleanup Datastore On Test Server
+
+    ${output}=  Run  bin/vic-machine-linux create --name=%{VCH-NAME} --target="%{TEST_USERNAME}:%{TEST_PASSWORD}@%{TEST_URL}" --thumbprint=%{TEST_THUMBPRINT} --image-store=%{TEST_DATASTORE} --bridge-network=bridge --container-network 'VM Network With Spaces': ${vicmachinetls}
+    Should Contain  ${output}  A network name must be supplied for port group VM Network With Spaces
+    Should Contain  ${output}  vic-machine-linux create failed
 
 Container network 1
-    Pass execution  Test not implemented
+    Set Test Environment Variables
+    # Attempt to cleanup old/canceled tests
+    Run Keyword And Ignore Error  Cleanup Dangling VMs On Test Server
+    Run Keyword And Ignore Error  Cleanup Datastore On Test Server
+
+    Log To Console  Create a portgroup with a space in it's name
+    ${out}=  Run  govc host.portgroup.add -vswitch vSwitchLAN 'VM Network With Spaces'
+
+    Log To Console  Create a bridge portgroup.
+    ${out}=  Run  govc host.portgroup.add -vswitch vSwitchLAN bridge
```

Comment:
can you use `--bridge-network=%{BRIDGE_NETWORK}` instead or is `bridge` required? Same in `Container network invalid 1`